src/renderers/ViewRenderer.js

```diff
@@ -106,7 +113,13 @@ class ViewRenderer extends SketchRenderer {
     const fill = makeColorFill(backgroundColor);
     const content = makeShapeGroup(frame, [shapeLayer], [fill]);
 
-    if (hasAnyDefined(style, SHADOW_STYLES)) {
+    if (props.shadowGroup) {
+      const shadows = [];
+      props.shadowGroup.map(shadowStyle =>
+        shadows.push(makeShadow(shadowStyle))
+      );
+      content.style.shadows = shadows;
+    } else if (hasAnyDefined(style, SHADOW_STYLES)) {
```

Comment:
instead of an `else if` we should probably concatenate the shadow defined in `style` if there is one